components/PlantForm.js

```diff
@@ -126,87 +126,41 @@ export default function PlantForm({
                 )}
               />
               {season.label}
-            </CheckboxLabel>
+            </label>
           ))}
         </OptionsContainer>
-      </Fieldset>
+      </fieldset>
 
       <StyledButton>
-        <button type="submit">
+        <ButtonSave type="submit">
           {isEditMode ? "Save Changes" : "Add Plant"}
-        </button>
-        <button type="button" onClick={onToggleForm}>
+        </ButtonSave>
+        <ButtonCancel type="button" onClick={onToggleForm}>
           Cancel
-        </button>
+        </ButtonCancel>
       </StyledButton>
-    </FormContainer>
+    </form>
   );
 }
 
-const FormContainer = styled.form`
-  border: 2px solid black;
-  border-radius: 8px;
-  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.1);
-  padding: 20px;
-  margin: 20px;
-  display: flex;
-  flex-direction: column;
-  gap: 20px;
-`;
-
-const Title = styled.h2`
-  font-size: 1.5rem;
-  margin-bottom: 10px;
-  color: #333;
-  display: flex;
-  justify-content: center;
-`;
-
-const FormLabel = styled.label`
-  display: flex;
-  flex-direction: column;
-  gap: 5px;
-  font-size: 1rem;
-  color: #555;
-`;
-
-const TextInput = styled.input`
-  padding: 8px;
-  border: 1px solid #ccc;
-  border-radius: 4px;
-  font-size: 1rem;
-  transition: border-color 0.3s ease;
+const ButtonCancel = styled.button`
+  background-color: var(--color-button-cancel);
 
-  &:focus {
-    border-color: #007bff;
-    outline: none;
+  &:hover {
+    background-color: var(--color-button-cancel-hover);
   }
 `;
 
-const TextArea = styled.textarea`
-  padding: 8px;
-  border: 1px solid #ccc;
-  border-radius: 4px;
-  font-size: 1rem;
-  transition: border-color 0.3s ease;
-  resize: none;
-
-  &:focus {
-    border-color: #007bff;
-    outline: none;
-  }
-`;
+const ButtonSave = styled.button`
+  background-color: var(--color-button-save);
 
-const Fieldset = styled.fieldset`
-  border: 1px solid #ddd;
-  border-radius: 4px;
-  padding: 10px;
+  &:hover {
+    background-color: var(--color-button-save-hover);
+  }
 `;
 
-const Legend = styled.legend`
-  font-size: 1rem;
-  color: #666;
-  padding: 0 5px;
+const FormLabel = styled.label`
```

Comment:
Consider adding `display: flex` to ensure that flex-box is properly working and the column direction is applied:
```suggestion
const FormLabel = styled.label`
  display: flex;
  flex-direction: column;
`;

```

Reply:
you would be also right here. But we defined `display: flex` in the styles.js for the `form`, so we wouldn't need it here